### DISCUSSION TITLE
How can I send and receive custom tokens using tronlink in shasta network

### DISCUSSION THREAD

**Discussion by derpsol** [community/community#57382] [Projects and Issues]
### Select Topic Area

Question

### Select Feature Area

- [ ] Issues
- [X] Projects

### Body

I want to add my custom tokens and NFTs in tronlink on shasta network.
First, I can't add such custom tokens in my wallet.
Second, I can add and send, receive tokens in nile network. If I can't add these in shasta network, how can I send and receive tokens in shasta network.
This is urgent problem and I wish there could be fast solution.
Thanks.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by Ernestarcos** [comment]
Deseo darme de baja que puedo Aser no se ni lo que ise

**Reply by ezappata** [reply]
Panties look awesome on me

**Reply by ezappata** [reply]
No they don't Raymond! R u gay
